hcipy/atmosphere/standard_atmosphere.py

```diff
@@ -12,3 +13,46 @@ def make_standard_atmospheric_layers(input_grid, L0=10):
 		layers.append(InfiniteAtmosphericLayer(input_grid, cn, L0, v, h, 2))
 
 	return layers
+
+def make_lco_atmospheric_layers(input_grid, r0=0.16, L0=25, wavelength=550e-9):
```

Comment:
No acronyms in function names. Maybe loose the "observatory" if you think it's too long?